static/app/views/alerts/rules/details/metricChart.tsx

```diff
@@ -135,7 +139,7 @@ function createIncidentSeries(
     trigger: 'item',
     alwaysShowContent: true,
     formatter: ({value, marker}) => {
-      const time = getFormattedDate(value, 'MMM D, YYYY LT');
+      const time = getFormattedDate(moment(value), 'MMM D, YYYY LT', {local: true});
```

Comment:
we might end up needing to check user config

Reply:
Added check for user timezone over default formatting